__tests__/app.test.js

```diff
@@ -82,3 +82,25 @@ describe("/api/articles/:article_id", () => {
    });
  });
 });
+describe("/api/articles", () => {
+ test("should return the correct status code and array of objects", () => {
+  return request(app)
+   .get("/api/articles")
+   .expect(200)
+   .then(({ body }) => {
+    expect(body.articles.length).toBe(13);
+    body.articles.forEach((article) => {
+     expect(article.body).toBe(undefined);
+     expect(typeof article.author).toBe("string");
```

Comment:
Have a look at a more specific jest matcher to use here to check that the object looks how you expect, instead of checking that the value is the right data type